requirements/edx/github.txt

```diff
@@ -48,7 +48,7 @@ git+https://github.com/edx/edx-lint.git@8bf82a32ecb8598c415413df66f5232ab8d974e9
 -e git+https://github.com/edx/xblock-utils.git@581ed636c862b286002bb9a3724cc883570eb54c#egg=xblock-utils
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 -e git+https://github.com/edx/edx-reverification-block.git@5da515ef229e73a137d366beb05ea4aea5881960#egg=edx-reverification-block
-git+https://github.com/edx/ecommerce-api-client.git@0.3.0#egg=ecommerce-api-client
+git+https://github.com/edx/ecommerce-api-client.git@0.3.0#egg=ecommerce-api-client==0.3.0
```

Comment:
@jimabramson the first time indicates what version of the repo to pull down, the second one indicates the version to pin the dependency to.  If you don't specify the pin version and a version already exists with the same egg name, it will not actually upgrade the dependency.  the `-e` installs circumvent this but have other bad side-effects.  We generally want to be moving towards not using `-e` and pinning the version explicitly.